data/spells/scripts/party/enchant.lua

```diff
@@ -10,9 +10,9 @@ condition:setParameter(CONDITION_PARAM_STAT_MAGICPOINTS, 1)
 condition:setParameter(CONDITION_PARAM_BUFF_SPELL, true)
 
 local baseMana = 120
```

Comment:
Hmm yeah it will, though I don't see if it is a performance hit at all. Keep it that way actually.